package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "mqtt",
   "description": "A library for the MQTT protocol",
-  "version": "1.5.0",
+  "version": "1.5.1",
```

Comment:
can you please roll this back to 1.5.0. The lead maintainer (myself) usually bump that number.